### ISSUE TITLE
[Bug]: [CN-DN BVT] show tables gives duplicate names if there are multiple account

### ISSUE THREAD

**Issue by aptend** [matrixorigin/matrixone#5813]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="776" alt="image" src="https://user-images.githubusercontent.com/49832303/197096461-bd9f8ed3-07cf-40da-a4b3-99a9d9e7116b.png">

**Comment by nnsgmsone** [comment]
need where accout_id = 0..

**Comment by nnsgmsone** [comment]
fixed by pr #5824
